### PR TITLE
Added layout for my Keyboardio Atreus

### PR DESCRIPTION
**Pull request by Xyverz**
Initial addition of my Keyboardio Atreus keyboard layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — CHANGES_REQUESTED** [review]
Looks good, except for the changes to the vusb submodule.

**Comment by Xyverz** [comment]
How do I remove that from the commit?

**Comment by dbast** [comment]
It comes via the second commit which merges the master branch.

One solution would be to rebase against master, e.g. `git rebase -i origin/master` ... or `git rebase -i upstream/master` ... depending on how you locally named the remote to the upstream repo (not your fork of it).

**Comment by Xyverz** [comment]
Removed the vusb submodule from the commits by rebasing as @dbast suggested. Thank you very much for that.

How do I undo the vusb submodule changes? I didn't mean to commit them and I don't really want them in my fork.

**Comment by dbast** [comment]
@fauxpark Can you have a look again? Tested locally and looks ready to merge :)

**Review by fauxpark — APPROVED** [review]

**Comment by Xyverz** [comment]
Whoops, I thought my PR had already been merged. Restored branch.

File: keyboards/keyboardio/atreus/keymaps/xyverz/keymap.c [hunk 1]
```diff
@@ -0,0 +1,191 @@
+// This is the personal keymap of Ian Sterling (@xyverz). It is based on the keymap by
```
Comment (drashna) [inline keyboards/keyboardio/atreus/keymaps/xyverz/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keyboardio/atreus/keymaps/xyverz/keymap.c (hunk 1 above)
Comment (Xyverz) [reply keyboards/keyboardio/atreus/keymaps/xyverz/keymap.c:1]: Done & pushed.

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks

**Pull request merged**